src/coprocessor/dag/expr/column.rs

```diff
@@ -38,8 +38,33 @@ impl Column {
     }
 
     #[inline]
-    pub fn eval_string<'a>(&self, row: &'a [Datum]) -> Result<Option<Cow<'a, [u8]>>> {
-        row[self.offset].as_string()
+    pub fn eval_string<'a>(
```

Comment:
what will happen if it is a binary char? will it be padded just as non-binary chars?

Reply:
YES. And you can have a try on mysql if you are not sure(https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sqlmode_pad_char_to_full_length)  @XuHuaiyu What do you think?